MapboxSpeech/MBSpeechOptions.swift

```diff
@@ -134,6 +170,13 @@ open class SpeechOptions: NSObject, NSSecureCoding {
      */
     @objc open var locale: Locale = Locale.autoupdatingCurrent
     
+    /**
+     Gender of voice speeking text.
+     
+     Note: not all languages have both genders.
+     */
+    @objc open var speechGender: SpeechGender = .female
```

Comment:
Instead of defaulting to female, define a new `neuter` case (along the lines of [`NSSpeechSynthesizer.VoiceGender.neuter`](https://developer.apple.com/documentation/appkit/nsspeechsynthesizer.voicegender/1448476-neuter)) that ends up not adding a `gender` query parameter to the URL. That way, the Voice API can prefer a male voice over a female voice in a particular language if the male voice happens to be of higher quality.